scss/_patterns_navigation.scss

```diff
@@ -550,75 +575,81 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
 
     display: none;
     margin: 0;
-    min-width: 100%;
+    width: 100%;
     padding: 0;
     z-index: 5;
 
     @media (min-width: $breakpoint-navigation-threshold) {
       position: absolute;
       // padding applied to .p-navigation__link + line-heigh of the anchor text inside
       top: $spv--large * 2 + map-get($line-heights, default-text);
+      width: auto;
     }
 
     @media (max-width: ($breakpoint-navigation-threshold - 1)) {
       box-shadow: none;
     }
   }
 
+  // allow nested dropdowns to be positioned correctly on larger screens
+  .p-navigation__dropdown {
+    .p-navigation__dropdown {
+      @media (min-width: $breakpoint-navigation-threshold) {
+        position: static;
+        top: 0;
+      }
+    }
+  }
+
   .p-navigation__dropdown--right {
     right: 0;
   }
 
   .p-navigation__item--dropdown-toggle {
-    position: relative;
-
-    &::after {
-      background-position: center;
-      background-repeat: no-repeat;
-      background-size: contain;
-      content: '';
-      display: block;
-      height: $spv--large;
-      pointer-events: none;
-      position: absolute;
-      right: map-get($grid-margin-widths, small);
-      text-indent: calc(100% + 10rem);
-      top: calc($spv--large + map-get($nudges, x-small));
-      transform: rotate(-90deg);
-      width: map-get($icon-sizes, default);
-
-      @media (min-width: $threshold-4-6-col) {
-        right: map-get($grid-margin-widths, default);
-      }
-
-      @media (min-width: $breakpoint-navigation-threshold) {
-        right: calc($sph--small + 1px); // 1px for the border on selects. this aligns it with any selects underneath
-        transform: rotate(0deg);
-      }
-    }
-
     &.is-active {
-      &::after {
+      > .p-navigation__link::after {
```

Comment:
Why are the direct child `>` selectors needed now?

Reply:
Since there are nested dropdown now, the style should only be applied to the immediate dropdowns and navigation links in the tree